extensions/amp-story/1.0/amp-story-page.js

```diff
@@ -43,6 +43,7 @@ import {
   expandableElementsSelectors,
 } from './amp-story-embedded-component';
 import {AnimationManager, hasAnimations} from './animation';
+import {CSS} from '../../../build/amp-story-page-attachment-0.1.css';
```

Comment:
You can do `{CSS as pageAttachmentCSS}` so you have a better variable name. If we start rendering multiple things in a shadow DOM in the story-page, we'll have to import multiple CSS eventually.